staff/daniel-villegas/pre-course/all-projects/project-3/scripts.js

```diff
@@ -0,0 +1,138 @@
+
+function bingo(){
+
+    var accountTurns = 0;
+    
+    
+    function getRandomNumber(){
+        return Math.floor(Math.random() * 10 +1);
```

Comment:
the output number can be repeated

Reply:
I'll try to think something for solve this!